Assets/Scripts/Visuals/UiBuilder/ContextSubmenuAtom.cs

```diff
@@ -58,10 +60,16 @@ public override void Update(Atom.Data newData)
 
                 var text = gameObject.gameObject.GetComponentInChildren<TextMeshProUGUI>();
                 text.text = newContextMenuTextData.title;
+                text.color = newContextMenuTextData.isDisabled ? Color.gray : Color.black;
 
                 var hoverHandler = gameObject.gameObject.GetComponent<ContextMenuHoverHandler>();
                 hoverHandler.OnHoverAction = () =>
                 {
+                    if (newContextMenuTextData.isDisabled)
```

Comment:
I think, the context menu should always show what it contains. The user should be able to see, what he can't do right now

Reply:
I understand that but wouldn't it also be nice for the user to not have to navigate through multiple submenus just to see that he can't do anything/or that there is one actual possible option? You really never have to know about options inside submenus like "addEntity" or "selectedEntity" unless you have selected an entity or are able to add an entity (?). But I'm sure there are also cases where you would want to know that an option exists even if you don't fulfill the prerequisites. I just thought it is easier to navigate in general.  I could add an optional parameter called "hideable", or just remove everything. (I could also just grey out the text but let it open for example. What do you think?

Reply:
I like the option of graying it out but still be able to open it